modyn/trainer_server/internal/trainer/remote_downsamplers/abstract_per_label_remote_downsample_strategy.py

```diff
@@ -6,17 +6,6 @@
 
 
 class AbstractPerLabelRemoteDownsamplingStrategy(AbstractRemoteDownsamplingStrategy):
-    def __init__(
-        self,
-        pipeline_id: int,
-        trigger_id: int,
-        batch_size: int,
-        params_from_selector: dict,
-        modyn_config: dict,
-        device: str,
-    ):
-        super().__init__(pipeline_id, trigger_id, batch_size, params_from_selector, modyn_config, device)
-        self.requires_data_label_by_label = True
```

Comment:
many subclasses still overwrite this field `requires_data_label_by_label` to `False`. Thus I leave it for subclass to decide. (actually only Craig is the only subclass that previous does not overwrite this field itself.)

Reply:
sounds good!